### DISCUSSION TITLE
plugin-block

### DISCUSSION THREAD

**Discussion by sytolk** [Milkdown/milkdown#828] [Q&A]
Hi, 
I have try [plugin block](https://milkdown.dev/plugin-block) and it works -> DnD move blocks of MD text. But there is not separator line that shown on start dragging 
![Screenshot 2022-11-16 at 10 47 26](https://user-images.githubusercontent.com/1415463/202136228-e47edfcf-fa39-4bcc-abea-3b8119bd1d2e.png)
The `<div style="z-index:50 background-color:.. "` is not attached to the DOM in my app. And I cannot found this div style in plugin-block source code. Where is comming from? and what can be the reason to not appear this block separator line for me??

**Comment by Saul-Mirone** [comment]
It's drop cursor introduced by  @milkdown/plugin-cursor

**Comment by sytolk** [comment]
Thanks! maybe documentation for plugin block its need to be updated: https://milkdown.dev/plugin-block I cant see dependency to @milkdown/plugin-cursor
